### DISCUSSION TITLE
控件目前只能全透明，希望能实现半透明功能

### DISCUSSION THREAD

**Discussion by xiaolangg** [Xiaokang2022/maliang#1] [Ideas]
当画布背景图片色彩较多较鲜艳是，全透明的控件有时候看不清文字，如果能设置透明度应该效果会很好。希望能实现此功能

**Comment by Xiaokang2022** [comment]
实际上，控件所谓的透明效果是通过颜色为空来实现的，也就是说，控件内部只能是单一颜色、图片或者不填充颜色的状态。当不填充颜色时，就时“透明”的。因此，基于此原理，目前暂时无法实现半透明的效果。

**Comment by TotoWang-hhh** [comment]
> 实际上，控件所谓的透明效果是通过颜色为空来实现的，也就是说，控件内部只能是单一颜色、图片或者不填充颜色的状态。当不填充颜色时，就时“透明”的。因此，基于此原理，目前暂时无法实现半透明的效果。

可以试试通过有间隔地增加透明像素来使其看上去如同半透明
